Yoast/YoastSEO.js
# Add premium synonyms styling

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Used flexbox for better usability (no more overlapping HTML blocks!)

## Test instructions

This PR can be tested by following these steps:

* Check out the example
* See a Premium checkbox
* See a Synonyms field
* Use the following text:
```
igor
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
wkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh iwkerklwejrliwejr oiwejr iowejr iowej riowejr iwelh aliueh luiweh i
jip
```
* Add a `igor` focus keyword
* Add a `jip` synonym
* Check the `Premium` box
* See the `Large parts of your text do not contain the keyword or its synonyms. Try to distribute them more evenly.` result.
* Use the eye-marker to see the Focus keyword -and- Synonym being marked

Fixes #1588
(Does not fix the nice-to-have)
